build.sbt

```diff
@@ -126,3 +126,46 @@ lazy val benchmarks = project
   .dependsOn(core.jvm)
   .settings(name := "lucuma-core-benchmarks")
   .enablePlugins(NoPublishPlugin, JmhPlugin)
+
+// for publishing CSS to npm
```

Comment:
I guess publishing scala.js to npm?

Reply:
Woops, copy-pasted this from lucuma-css :)